xray/test/test_backends.py

```diff
@@ -196,6 +196,11 @@ def test_roundtrip_timedelta_data(self):
         with self.roundtrip(expected) as actual:
             self.assertDatasetIdentical(expected, actual)
 
+    def test_roundtrip_float64_data(self):
+        expected = Dataset({'x': ('x', np.array([1.0, 2.0], dtype='float64'))})
```

Comment:
Can we use something with more digits here? Perhaps `np.pi`? Otherwise this looks good.